docs/user/user-defined-redirects.rst

```diff
@@ -25,19 +25,17 @@ the bad user experience of a 404 page is usually best to avoid.
      A guide to deprecating features and other topics in a documentation.
 
 
-Special features
-----------------
-
-- By default, redirects are followed only if the requested page doesn't exist
-  (*404 File Not Found* error).
-  If you need to apply a redirect for files that exist,
-  **This option is only available on some plan levels**.
-  Please ask support if you need it for some reason.
-- :ref:`user-defined-redirects:page redirects` and :ref:`user-defined-redirects:exact redirects`
+Limitations
+-----------
+
+- By default, redirects only apply on pages that don't exist.
+  **Forced Redirects** allow you to apply redirects on existing pages, 
+  but incur a small performance penalty, so aren't enabled by default.
+  You can ask for them to be enabled via support.
```

Comment:
Maybe it's worth linking to our page on getting support: https://docs.readthedocs.io/en/stable/support.html

Reply:
Also, that support page needed some updates, I threw up some changes here: https://github.com/readthedocs/readthedocs.org/pull/10580